solitude/exceptions.py

```diff
@@ -2,9 +2,18 @@
 from rest_framework.views import exception_handler
 
 from lib.bango.errors import BangoImmediateError
+from lib.brains.errors import BraintreeResultError
 
 
 def custom_exception_handler(exc):
+    if isinstance(exc, BraintreeResultError):
+        res = {
+            '__all__': [exc.result.message],
+            '__braintree__': 'error',
+            '__type__': 'braintree'
```

Comment:
That's what we've got in for some errors right now, but our errors are really inconsistent, so I filed https://github.com/mozilla/solitude/issues/349 to go through, unfortunately things in webpay and zamboni depend on formats, so it will be a bit of a hairy change.